src/components/Badge/__stories__/Badge.stories.mdx

```diff
@@ -0,0 +1,213 @@
+
+<Canvas>
+  <Story name="Outbound">
+    <Flex direction={Flex.directions.COLUMN} gap={Flex.gaps.MEDIUM} align={Flex.align.START}>
+      <Badge outbound>
+        <Link text="Read more" />
+      </Badge>
+      <Badge outbound>
+        <Link text="What's new" iconPosition={Link.iconPositions.START} icon={WhatsNew} />
+      </Badge>
+      <Badge outbound>
+        <Link text="Learn more" iconPosition={Link.iconPositions.END} icon={ExternalPage} />
+      </Badge>
+    </Flex>
+  </Story>
+</Canvas>
+
+### Positioning
+
+We can set the position of the badge in any of the edges of its child component
+
+<Canvas>
+  <Story name="Positioning">
+    <Flex direction={Flex.directions.COLUMN} gap={Flex.gaps.LARGE}>
+      <Flex gap={Flex.gaps.LARGE}>
+        <Badge circular type={Badge.types.COUNTER} position={Badge.positions.TOP_START}>
+          <Avatar size={Avatar.sizes.LARGE} src={person} type={Avatar.types.IMG} />
+        </Badge>
+        <Badge circular type={Badge.types.COUNTER} position={Badge.positions.TOP_END}>
+          <Avatar size={Avatar.sizes.LARGE} src={person} type={Avatar.types.IMG} />
+        </Badge>
+      </Flex>
+      <Flex gap={Flex.gaps.LARGE}>
+        <Badge circular type={Badge.types.COUNTER} position={Badge.positions.BOTTOM_START}>
+          <Avatar size={Avatar.sizes.LARGE} src={person} type={Avatar.types.IMG} />
+        </Badge>
+        <Badge circular type={Badge.types.COUNTER} position={Badge.positions.BOTTOM_END}>
+          <Avatar size={Avatar.sizes.LARGE} src={person} type={Avatar.types.IMG} />
+        </Badge>
+      </Flex>
+    </Flex>
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component: (
+          <Badge outbound>
+            <div
+              style={{
+                border: "1px dashed var(--ui-border-color)",
+                borderLeftColor: "transparent",
+                borderBottomColor: "transparent"
+              }}
+            >
+              <Link text="Read more" />
+            </div>
+          </Badge>
+        ),
+        description: "When using the badge on an inline component, apply 'outbound' to it"
+      },
+      negative: {
+        component: (
+          <Badge>
+            <div
+              style={{
+                border: "1px dashed var(--ui-border-color)",
+                borderLeftColor: "transparent",
+                borderBottomColor: "transparent"
+              }}
+            >
+              <Link text="Read more" />
+            </div>
+          </Badge>
+        ),
+        description: "Do not leave the indicator inside the element boundaries"
+      }
+    },
+    {
+      positive: {
+        component: (
+          <Badge color={Badge.colors.PRIMARY} border>
+            <Button leftIcon={WhatsNew}>What's new</Button>
+          </Badge>
+        ),
+        description: "When using badge with same color as the child component, use 'border' prop"
```

Comment:
I actually wonder why this isn't the default. it looks much better even in different colors

with border
<img width="476" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/6093192/cde78b45-c6c8-43fd-975b-811617558d16">

without
<img width="452" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/6093192/177bc74b-2ed3-412c-8620-fa89da613c32">

Reply:
I actually agree on that :)
I'll take it with Yael or Meirav